kong/tools/stream_api.lua

```diff
@@ -0,0 +1,114 @@
+
+local stream_api = {}
+
+local _endpoints = {}
+
+
+function stream_api.register(t)
+  for path, handler in pairs(t) do
```

Comment:
We should consider caching these Lua built-in functions at module level.

Reply:
done.  (this one happens only at initialization, but there are others that are part of the request path, so better to do all of them)